webapp/package-lock.json

```diff
@@ -5851,7 +5851,8 @@
     },
     "node_modules/classnames": {
       "version": "2.5.1",
-      "license": "MIT"
+      "resolved": "https://registry.npmjs.org/classnames/-/classnames-2.5.1.tgz",
+      "integrity": "sha512-saHYOzhIQs6wy2sVxTM6bUDsQO4F50V9RQ22qBpEdCW+I+/Wmke2HOl6lS6dTpdxVhb88/I6+Hs+438c3lfUow=="
```

Comment:
what changed this

Reply:
I might have used incorrect npm version previously and this fix might have been caused by using the right version again. But not sure though.